core/modules/system/system.install

```diff
@@ -589,13 +589,13 @@ function system_requirements($phase) {
 
   if ($phase == 'runtime') {
     // Inform the user if theme_debug is enabled.
-    $theme_debug_enabled = config_get('system.core', 'theme_debug');
+    $theme_debug_enabled = settings_get('theme_debug', config_get('system.core', 'theme_debug'));
     if ($theme_debug_enabled) {
       $requirements['theme_debug'] = array(
         'title' => $t('Theme debug'),
         'value' => $t('Enabled'),
         'severity' => REQUIREMENT_WARNING,
-        'description' => $t('Theme debugging is currently enabled, and should be disabled on a live site. This setting can be changed using the <a href="!url">Devel module</a>.', array('!url' => module_exists('devel') ? url('admin/config/development/devel') : 'https://backdropcms.org/project/devel')),
+        'description' => $t('Theme debugging is currently enabled, and should be disabled on a live site. This setting can be changed using the <a href="!url">Devel module</a> or, if set in the settings.php file, by removing it from the settings.php file.', array('!url' => module_exists('devel') ? url('admin/config/development/devel') : 'https://backdropcms.org/project/devel')),
```

Comment:
See previous comment, re conditionally adding help text about where this setting has been configured:

```suggestion
        'description' => $theme_debug_description,
```

Reply:
I think this is a valid change. Though I think to help keep this PR slim let's put all the messaging changes to the follow-up issue. I'd like to implement this slightly differently by coming up with a better way to know if something is set in the global $settings (a new function).